AATAMS/AATAMS_acoustic/check_csv.py

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+"""
+Check that csv file has required fields 
+"""
+import os
+import sys
+import pandas
+
+csvfile = sys.argv[1]
+header_col = ['transmitter_id', 'installation_name', 'station_name', 'receiver_name', 'detection_timestamp', 'longitude', 'latitude',
+              'sensor_value', 'sensor_unit', 'FDA_QC', 'Velocity_QC', 'Distance_QC', 'DetectionDistribution_QC', 'DistanceRelease_QC',
+              'ReleaseDate_QC', 'ReleaseLocation_QC', 'Detection_QC']
+
+# CSV file to extract array of formatted data
+data = pandas.read_csv(csvfile, delimiter=';', header=0)
+
+if set(data.columns.values).issubset(header_col) == False:
```

Comment:
According to the [docs](https://docs.python.org/2/library/stdtypes.html#set.issubset), this is testing that every column in the data is in `header_col`, not the other way around. From the error message below, it looks like you actually want
```python
if not set(header_col).issubset(data.columns.values):
```
i.e. to check that every that every element of `header_col` is present in the data!

Or, the columns in the data should be exactly the ones listed in `header_col`, with no additional ones, then simply
```python
if not set(data.columns.values) == set(header_col):
```